webservices/args.py

```diff
@@ -716,6 +716,7 @@ def make_seek_args(field=fields.Int, description=None):
     'max_debts_owed_by_committee': Currency(description='Maximum debt'),
     'federal_funds_flag': fields.Bool(description=docs.FEDERAL_FUNDS_FLAG),
     'has_raised_funds': fields.Bool(description=docs.HAS_RAISED_FUNDS),
+    'active_candidates': fields.Bool(missing=True, description=docs.ACTIVE_CANDIDATE),
```

Comment:
Defaulting to only show active candidates is a breaking change - we should keep the existing behavior the same and include all candidates (active and inactive) by default.

Reply:
that is something we have discussed and confirmed with Paul - we'll keep the current change for now.. Thx

Reply:
Thanks @qqss88. If this was a new feature, that might be fine, but the /candidates/totals/ endpoint feeds into the candidate datatables on the front end, and the endpoint has been around for a while. Changing expected behavior is a breaking change, so we shouldn't provide a default value and hide inactive candidates. I confirmed this with @PaulClark2.